torch/ao/quantization/fx/utils.py

```diff
@@ -642,3 +653,51 @@ def get_skipped_module_name_and_classes(
         skipped_module_classes += get_custom_module_class_keys(prepare_custom_config.float_to_observed_mapping)
 
     return skipped_module_names, skipped_module_classes
+
+def _validate_qconfig_against_dtype_config(
+        qconfig: QConfigAny,
+        dtype_config: DTypeConfig,
+        pattern: Pattern):
+    """
+    Validate whether the QConfig satisfies all of the following constraints from the backend:
+        1. QConfig specified a quantization range that falls within the backend's, if any
+        2. QConfig specified a min scale value that is >= the backend's min scale value, if any
+    """
+    def validate_activation_post_process(
+            activation_post_process_ctr: Union[_PartialWrapper, Type[ObserverBase]],
```

Comment:
might be good to always pass an observer here, instead of the constructor.  Passing functions around adds complexity and makes it harder to think about the code.  Creating a new observer is cheap.